pallets/vc-management/src/lib.rs

```diff
@@ -44,8 +44,8 @@ pub use assertion::*;
 mod schema;
 pub use schema::*;
 
-// VCID type in the registry, maybe we want a "did:...." format?
-pub type VCID = u64;
+// hash of a VC, computed via blake2_256
+pub type VCHASH = H256;
```

Comment:
Maybe I need to get this, but after the user gets a VC, how can he verify the VC hash without a unique ID? He can only tell the hash of the VC exists on chain, but not necessarily **match** the VC he gets, isn't it 🤔

Reply:
Anyway, the VCID and VCHASH are both needed in the VCMP. Fixing this right now.